tests/test_base.py

```diff
@@ -13,15 +13,14 @@ def test_base_creation(example_model):
     model, crs = example_model
     dummy = DummyObj(model=model, title=title)
 
-    # UUID should exist, but not root or part
+    # UUID and part should exist, but not root (yet)
```

Comment:
More strictly, "UUID should exist, and part (name) should be determinable, but root should not exist yet."

Most of the time we can use the term 'part' rather lazily to mean either the string that is a part name – whether or not that part has come into full existence and been added to a Model – or to mean a fully added part which has the part name as its key in the Model's internal dictionary. The base class part is providing the first of these.